pkg/gitops/fileprocessor/gotemplates.go

```diff
@@ -0,0 +1,60 @@
+package fileprocessor
+
+import (
+	"bytes"
+	"github.com/kris-nova/logger"
+	"github.com/pkg/errors"
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"strings"
+	"text/template"
+)
+
+const (
+	templateExtension = ".tmpl"
+)
+
+// TemplateParameters represents the API variables that can be used to template a profile. This set of variables will
+// be applied to the go template files found. Templates filenames must end in .templ
+type TemplateParameters struct {
+	ClusterName string
+}
+
+// NewTemplateParameters creates a set of variables for templating given a ClusterConfig object
+func NewTemplateParameters(clusterConfig *api.ClusterConfig) TemplateParameters {
+	return TemplateParameters{
+		ClusterName: clusterConfig.Metadata.Name,
+	}
+}
+
+// GoTemplateProcessor is a FileProcessor that executes Go Templates
+type GoTemplateProcessor struct {
+	Params TemplateParameters
+}
+
+// ProcessFile takes a template file and executes the template applying the TemplateParameters
+func (p *GoTemplateProcessor) ProcessFile(file File) (*File, error) {
+	if !isGoTemplate(file.Path) {
+		logger.Debug("copying non template file %q", file.Path)
```

Comment:
I know this is just a logging statement, but `GoTemplateProcessor` isn't aware that the files are being copied. Maybe this should read something along these lines:
```suggestion
		logger.Debug("leaving non-template file unmodified %q", file.Path)
```

Reply:
Good point, changing